### PR TITLE
Installation products

### PR DESCRIPTION
**Pull request by EvanBarbour3**
## Features
- Added available products for an installation tab to installation view

### CODE REVIEW

File: app/Http/Controllers/InstallationsController.php [hunk 1]
```diff
@@ -166,4 +169,46 @@ private function fetchInstallation($id)
     {
         return $this->fetchModelByIdWithMerchantLimit((new Installation()), $id, 'installation', '/installations');
     }
+
+    /**
+     * @author EB
+     * @param int $id
+     * @return array
+     */
+    private function fetchProducts($id)
```
Comment (sped2773) [inline app/Http/Controllers/InstallationsController.php]: Feel controller should be dumb and gateway should return listProductGroupWithProducts with the API doing the heavy lifting incase merchants do a deeper integration and bot use our back office. Raise with Woj and we can chat through if needed.

Comment (wnowicki) [inline app/Http/Controllers/InstallationsController.php]: Agree. Not a place for this code. Move to Service or Gateway.

Comment (wnowicki) [inline app/Http/Controllers/InstallationsController.php]: Also. Yes, you can do maximum two API calls here! :wink:

Comment (wnowicki) [inline app/Http/Controllers/InstallationsController.php]: Second option. Add to Merchant API:

`GET /v4/installations/:installation/product-groups/:product-group?with=products` and put logic in Merchant API.

**Pull request merged**